packages/report-server/src/__tests__/reportBuilder/transform/mergeRows.test.ts

```diff
@@ -46,7 +46,11 @@ describe('mergeRows', () => {
         },
       },
     ]);
-    expect(transform(MERGEABLE_ANALYTICS)).toEqual([{ BCD1: 28, BCD2: 123 }]);
+    expect(transform(TransformTable.fromRows(MERGEABLE_ANALYTICS))).toEqual(
+      TransformTable.fromRows([
+        { period: undefined, organisationUnit: undefined, BCD1: 28, BCD2: 123 },
+      ]),
+    );
   });
 
```

Comment:
As we have undefined value, suggest to use `toStrictEqual` instead of `toEqual`. Difference: https://jestjs.io/docs/expect#tostrictequalvalue

Also I think we should use `toStrictEqual` in other unit tests to ensure that transforms don't add unexpected undefined value. 

As we are refactoring all unit tests in this PR, should be a good chance to switch?

Reply:
In this specific case, I added `undefined` values more to just get the columns to appear in the correct order then to validate the data.

I think I'm generally a bit torn on using `toStrictEqual`. It would be more accurate, but it might make the tests more painful to write/maintain if there are frequent `undefined` values (which I suspect there might be...)

I'll try switching and see how things go 👍

Reply:
Thanks for switching that🙏